.env

```
@@ -1 +1,5 @@
 DATABASE_URL="postgresql://user:password@host:port/db"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)